libminifi/test/unit/MinifiConcurrentQueueTests.cpp

```diff
@@ -31,6 +31,20 @@ namespace utils = org::apache::nifi::minifi::utils;
 
 namespace {
 
+  template <typename Function, typename Duration>
```

Comment:
I think relying on chrono::duration is a bit more bulletproof:
```
template<typename Function, class Rep, class Period>
bool becomesTrueWithinTimeout(const Function &conditio, const std::chrono::duration<Rep, Period>& timeout) {
```

Reply:
I was mainly trying to shave off a few characters, but yeah, it's probably not worth it.  I have changed the argument type to `std::chrono::duration<Rep, Period>` but kept it as pass-by-value, since it is just a number.

Reply:
Okay, thanks!